rejson_test.go

```diff
@@ -2,31 +2,229 @@ package rejson
 
 import (
 	"encoding/json"
+	"github.com/Shivam010/go-rejson/rjs"
 	"reflect"
 	"testing"
 
-	"github.com/gomodule/redigo/redis"
+	goredis "github.com/go-redis/redis"
+	redigo "github.com/gomodule/redigo/redis"
 )
 
-type TestObject struct {
-	Name   string `json:"name"`
-	Number int    `json:"number"`
+func TestUnsupportedCommand(t *testing.T) {
+	_, _, err := rjs.CommandBuilder(1234, nil)
+	if err == nil {
+		t.Errorf("TestUnsupportedCommand() returned nil error")
+		return
+	}
 }
 
-func TestJSONSet(t *testing.T) {
-	conn, err := redis.Dial("tcp", ":6379")
+func TestRedigo(t *testing.T) {
```

Comment:
I really like how you structured this ! 💯

Reply:
Thanks 😅